src/main/java/com/datapipe/jenkins/vault/credentials/common/VaultCertificateCredentialsImpl.java

```diff
@@ -132,28 +132,30 @@ private static String unwrap(@NonNull String wrapped) {
 
     @Extension
     public static class DescriptorImpl extends BaseStandardCredentialsDescriptor {
+        public static final String defaultKeyStoreKey = DEFAULT_KEYSTORE_KEY;
```

Comment:
I suggest we move the DEFAULT_KEYSTORE_KEY and DEFAULT_PASSWORD_KEY to the DescriptorImpl

So they can be referred to in jelly.

and the VaultCertificateCredentialsImpl can also just refer to them from the DescriptorImpl

Reply:
Done too